### DISCUSSION TITLE
getting information about the obstacles in the enviornment

### DISCUSSION THREAD

**Discussion by abdelhakim96** [microsoft/AirSim#3652] [Support Q&A]
I am trying to make a python script that makes the drone move towards the obstacles in the environment. Is there a way that i can obtain the obstacles coordinates without inputting it manually?

**Comment by servesh-chaturvedi** [comment]
Hey there, you could watch this from around 37 min to see if it helps. Tells about the relevant APIs and other insights.
https://youtu.be/SvwjrPQWbng

**Comment by jonyMarino** [comment]
Hi and welcome @abdelhakim96! You can get that info from the voxel grid generator: https://microsoft.github.io/AirSim/voxel_grid/
